src/index.js

```diff
@@ -24,6 +25,11 @@ NAF.schemas.add({
   ]
 });
 
+NAF.schemas.add({
+  template: "#hand-template",
+  components: ["position", "rotation", "visible"]
```

Comment:
This is what I am liking about NAF so far, the networking has no idea about how this visible component is getting updated, and the `hand-controls-child-visibility` component has no idea its dealing with networking.

Reply:
Yeah, it's holding up really well so far!